Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -1302,4 +1321,23 @@ namespace O3DE::ProjectManager
 
         return result && updateAvaliableResult;
     }
+
+    AZStd::pair<AZStd::string, AZStd::string> PythonBindings::GetSimpleDetailedErrorPair()
+    {
+        AZStd::string detailedString = m_pythonErrorStrings.size() == 1
+            ? ""
+            : AZStd::accumulate(m_pythonErrorStrings.begin(), m_pythonErrorStrings.end(), AZStd::string(""));
```

Comment:
Do all these errors end with a newline?

Otherwise they would be slammed right next to each other without any whitespace

Reply:
Yes, they all end with a newline